server.js

```diff
@@ -0,0 +1,377 @@
+const http = require('http');
```

Comment:
I don't understand why you created this file at the root rather than using the `server.js` in the `app/` directory. It's a bit confusing but I suppose it works just the same.